### DISCUSSION TITLE
incorrect charecter detect

### DISCUSSION THREAD

**Discussion by shgh54** [jsvine/pdfplumber#676] [Ask for help with specific PDFs]
Hi all,
I have a scanned pdf file and I am using the pdfplumber to extract lines.
the issue is it wrong detect character "B" and show "8".
how can I fix the issue?

Thank you for your help.

**Comment by jsvine** [comment]
Hi @shgh54, unfortunately this sounds like a problem with whatever tool was used to run OCR (optical character recognition) on the scan, and is not related to `pdfplumber`. You can try using another OCR tool to reprocess it, though the success will depend on the quality of the scan.
